en/people/fabio-fumarola.md

```diff
@@ -0,0 +1,27 @@
+role: Machine Learning Engineer
+is_new: true
+twitter_user: fabiofumarola
+medium_user:
+linkedin_url: http://linkedin.com/in/fabio-fumarola
+ref: fabio-fumarola
+parent_ref: team
+start_date: October 30, 2017
+period_provided: until September 16, 2018
+annual_compensation: € 90,000
+approved_by_court: -
+---
+I was eight years old when I started to program with a Commodore VIC-20 bought together with an English course from my parents, but it was all by chance. Finished high school, I decided to study computer science at University of Bari. At the beginning it was hard, but after that I literally fallen in love for this field of study. I got a first and second level degree in five years, in 2008 I received a prize as best graduate student for the Science Faculties at University of Bari, and decide to go for a PhD in Machine Learning.
+
+During the doctorate, I learned how to make research. It is a mix of study of the literature by reading papers, look ahead thinking, creativity, passion and hard work (around 70%).I enjoyed to collaborate with a lot of researchers around the world, and to spend several months as invited researcher at the Department of Computer Science of the University of Illinois working with the Professor Jiawei Han.
+During this period I did research on Information Network Analysis with Tim Weninger and produced great research published on top ranked journal and conferences. In May 2011 I defended my thesis with supervisor Professor Donato Malerba. Got the Ph.D, I spent one year at the the Department of Biomedical Science and Human Oncology as research assistant, where I collaborated on the research to identify the genes that are modified by blood tumors. Simultaneously, I was research assistant at University of Bari working on Data Stream Mining applied to photo-voltaic plants. In 2013, I worked as Data Scientist at Mer.Mec. group and founded an Innovative Startup with the goal of applying Machine Learning algorithms to create map that helps in monitoring the quality of city neighborhoods. In 2014, I was research assistant at University of Bari working on Web Mining and Big Data.
+
+In 2015, I started a position as Data Scientist in Unicredit Research and Development. I spent almost three years there working on a lot of interesting stuffs ranging from Natural Language Processing, Graph Mining, Big Data, data encryption and High Frequency Trading. I was a great experience with wonderful and open minded researchers.
+
+During 2017, I heard about the Digital Transformation Team while I was in Unicredit. It was something that I followed with a lot of interest. Then, in late 2017 I found that they had an open position as Machine Learning Engineer and I sent my curriculum, and after several interviews I got hired. I'm very honored to work in the Digital Transformation Team under the guidance of Diego Piacentini. I'm sure that I'll learn a lot from him and from the team and to collaborate in driving innovation to my Country. 
```

Comment:
if the main verb is 'learn', 'to collaborate in' doesn't sound good to me. I'd rephrase as follows:
I'm sure that I'll learn a lot from him and from the team, as well will get the chance to contribute in driving innovation to my Country.